From ca7be9a919daad1fd6a1a5e1c5de5c9d36560de9 Mon Sep 17 00:00:00 2001
From: Erik Bernoth <erik.bernoth@googlemail.com>
Date: Tue, 17 Jul 2012 17:15:04 +0200
Subject: [PATCH] connman: do not allow connman to be DNSPROXY

    When conman tries to be a DNS proxy, many services can't use DNS for
    a still unknown reason. This workaround makes connman write it's
    knowledge to /etc/resolv.conf which can be understood by most services.

Signed-off-by: Erik Bernoth <erik.bernoth@googlemail.com>
---
 src/connman.service.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/connman.service.in b/src/connman.service.in
index a234a0c..2624678 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -5,7 +5,7 @@ After=syslog.target
 [Service]
 Type=dbus
 BusName=net.connman
-ExecStart=@prefix@/sbin/connmand -n
+ExecStart=@prefix@/sbin/connmand -n -r
 
 [Install]
 WantedBy=multi-user.target
-- 
1.7.7

